File: EducationApp.DataAccessLayer/Models/Filters/FilterUserModel.cs
using EducationApp.DataAccessLayer.Entities.Enums;
using EducationApp.DataAccessLayer.Models.Filters.Base;

namespace EducationApp.DataAccessLayer.Models.Filters
{
    public class FilterUserModel : BaseFilterModel
    {
        public Enums.BlockState IsBlocked { get; set; }
    }
}

File: AngularEducationStore/src/app/shared/models/filter/filter-user-model.ts
import { BaseFilterModel } from 'src/app/shared/models/filter/base/base-filter-model';
import { PageSize } from 'src/app/shared/enums/page-size';
import { IsBlocked } from 'src/app/shared/enums/is-blocked';

export class FilterUserModel extends BaseFilterModel {
    isBlocked: IsBlocked;
}
